label_studio/ml/examples/mmdetection/mmdetection.py

```diff
@@ -3,9 +3,9 @@
 
 from mmdet.apis import init_detector, inference_detector
 
-from label_studio.ml import LabelStudioMLBase
```

Comment:
`ml/examples` dir should be totally removed from repo, made in another branch but could be done here instead

Reply:
I'll do it here